texastribune/donations
# Build portal navbar using design system

#### What's this PR do?
+ Creates a navigation Vue component
+ Styles it with the DS `c-navbar` partial (see [related PR](https://github.com/texastribune/ds-toolbox/pull/26))

#### Why are we doing this? How does it help us?
An app is not complete without navigation!

#### How should this be manually tested?
+ `make`
+ `make restart`
+ Go to `/user-portal`
+ Confirm there's a white-background nav that has a functioning dropdown on small screens

#### How should this change be communicated to end users?
NA.

#### Are there any smells or added technical debt to note?
Less! Cleaning up the nav CSS felt good.

#### What are the relevant tickets?
UMP.

#### Have you done the following, if applicable:
* [x] Updated the documentation/wiki? *(added comments in DS repo)*
